### DISCUSSION TITLE
Missing tiles or mixed tiles

### DISCUSSION THREAD

**Discussion by PekoStudioSRO** [mapsforge/vtm#822] [Q&A] [answered]
Hi,
we implement mapsforge/vtm library in our project and when using online vector maps, we get sometimes in one zoom (100 m ) missing tiles - there are white spaces.
And sometimes we get a mixed tiled - I attach attachement to show you. Do you have any idea where the problem can be?

Thank you!

![Screenshot_20210322_152418_cz vnt dogtrace gps test](https://user-images.githubusercontent.com/34579991/113110111-01597b80-9207-11eb-8388-4b02ce646328.jpg)
<img width="408" alt="Snímek obrazovky 2021-03-31 v 9 56 14" src="https://user-images.githubusercontent.com/34579991/113110419-57c6ba00-9207-11eb-8c2e-7f09bb5a1230.png">

**Comment by devemux86** [comment, marked as answer]
What online map?

There are several samples in vtm-android-example which seem to work fine.
I cannot know your implementation, but you could compare your code with the samples.
